LICENSE

```diff
@@ -186,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2014-2017 Google Inc.
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
Oops.

Reply:
No, it turns out this is how the license file is supposed to appear.

Reply:
Attribution implied by the publisher?

Reply:
Oh, I see now. This is the bottom of the file talking about how to attribute yourself when you copy the files. *I* was copywrong.

Reply:
It was news to me too. I learned about this during some discussions with the folks who own https://opensource.google.com/docs/releasing/#checklist